CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## 0.2.0
+Released on August 12th, 2022.
```

Comment:
```suggestion
Released on August 15th, 2022.
```

Reply:
Can we also add a line describing the breaking change that we made and what code updates need to be made?